specs/protocol/isthmus/exec-engine.md

```diff
@@ -0,0 +1,45 @@
+# L2 Execution Engine
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Fees](#fees)
+  - [Configurable fees](#configurable-fees)
+    - [Configuring scalars](#configuring-scalars)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Fees
+
+New OP stack variants have different resource consumption patterns, and thus require a more flexible
+pricing model. To enable more customizable fee structures, Isthmus adds a new component to the fee
+calculation: the `ConfigurableFee`, which is parameterized by two scalars: the `configurableFeeScalar`
+and the `configurableFeeConstant`.
+
+### Configurable fees
+
+The configurable fee is set as follows:
+
+`configurableFee = gas_used * configurableFeeScalar + configurableFeeConstant`
```

Comment:
So we don't need any fractional scaling, like we introduced with Fjord for the model parameters? I mean something like
```suggestion
`configurableFee = (gas_used * configurableFeeScalar + configurableFeeConstant) / 1e6`
```
to allow for a decimal precision of 6.

Reply:
Good point -- it makes sense for users to be able to have fractional scalars. However, I don't know why a user would want to have a fractional constant. The only reason I can think would be to save bits -- see my other comment.